layouts/maternity-care-sharing-intro.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Foobar</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700%7cSource+Serif+Pro:400,600" rel="stylesheet">
+	<link rel="stylesheet" type="text/css" href="https://samaritanministries.org/stylesheets/application.css">
+
+	<link rel="stylesheet" type="text/css" href="/css/main.css">
+	<link rel="stylesheet" type="text/css" href="/css/layouts/maternity-care-sharing-intro.css">
+</head>
+
+<body>
+	<div class="content">
+
+		<section class="maternity-care-sharing-intro">
+			<div class="maternity-care-sharing-intro__container">
+				
+				<h2 class="maternity-care-sharing-intro__header">One-Person memberships<br />must choose one of the following options <strong>prior to conception</strong> for a maternity need to be shared:</h2>
+
+				<p class="maternity-care-sharing-intro__option-header">Option 1</p>
```

Comment:
Should this be a sub component?

Its overkill for a simple component like this but will be a lifesaver with a more complex component.

```

    <section class="maternity-care-sharing-intro">
      <div class="maternity-care-sharing-intro__container">
        
        <h2 class="maternity-care-sharing-intro__header"></h2>
        
        <div class="maternity-care-sharing-intro__option">
          
          <div class="maternity-care-option">
            <h3 class="maternity-care-option__header"></h3>
            <h4 class="maternity-care-option__sub-header"></h4>
            <p class="maternity-care-option__description"></p>
          </div>
          
        </div>
        
        <div class="maternity-care-sharing-intro__option">
          
          <div class="maternity-care-option">
            <h3 class="maternity-care-option__header"></h3>
            <h4 class="maternity-care-option__sub-header"></h4>
            <p class="maternity-care-option__description"></p>
          </div>
          
        </div>

        <p class="maternity-care-sharing-intro__guidelines"></p>

        <p class="maternity-care-sharing-intro__button-container">
          <a href="#" class="maternity-care-sharing-intro__button"></a>
        </p>
        
      </div>
    </section>
```

This is not a must have if you're bumping against estimate.

Reply:
Bios layout might be a good to look at to see sub-components used in a simple layout